### ISSUE TITLE
Suggestions: Changing Alex's Mobs.

### ISSUE THREAD

**Issue by LocoPineapples** [VazkiiMods/crucial2#223]
I feel like as it is in the modpack currently, Alex's Mobs isn't living up to its full potential. I'm not sure why some mobs were disabled, whether it was for lag or for balance, but not having all the mobs available really makes it feel like a shell of what it could be. While playing, I constantly noticed and missed a lot of the animals that weren't there.

For starters, many of the animals are made with the same intent Mojang makes them: to introduce players to some unique and wonderful animals that we share our world with, and to translate them into game in a respectful manner. It allows players to learn about some rare and endangered creatures that they wouldn't otherwise know about! As someone who works with animals, this is extremely important in order to foster passion for the natural world. Players are encouraged to work with the animals in a unique way to get their resources, or use their behavior to their advantage, as opposed to killing them.  These uses allow players much more freedom in their builds, from using Phages and Crows to automate, to creating natural defenses with Crocodiles and Snapping Turtles, or even just having a friendly group of seals helping you dredge items from the bottom of the seafloor. This isn't hurt at all by the fact that this mod is extremely good quality.

I appreciate you reading this! If you don't add the rest of the mobs, I understand, and appreciate you making the pack at all!

**Comment by CheetahCyberBoi** [comment]
I very much agree

**Comment by PRADDD** [comment]
agreed. a lot of mobs from this mod are disabled and I feel like they can be enabled without breaking the feel that crucial has.

**Comment by megatrook** [comment]
i disagree. most of the mobs disabled should stay disabled. i could see a couple more being re-enabled but i honestly feel that the choices for this pack are really good and well-chosen

**Comment by QuietObserver** [comment]
I agree with adding more of alex's mobs, here is what I could see enabled:

Bald Eagle - has some very cool uses, swooping down to attack mobs and allowing you to see hard to reach areas

Blobfish - doesn't have too much of a use besides memeing and taming bald eagles, but a very nice aesthetic change

Cave centipede - a new hostile mob in the underground would be great. The forgotten already does this, but I feel like cave centipedes could be a bit more common, and they also add decent functionality and challenge

Cockroach - mainly aesthetic, not too much functionality but could make the underground more varying and could make cave roots more challenging to use(because they are very OP at the moment)
